cql3/attributes.cc

```diff
@@ -123,7 +122,7 @@ db::timeout_clock::duration attributes::get_timeout(const query_options& options
     if (timeout.is_null() || timeout.is_unset_value()) {
         throw exceptions::invalid_request_exception("Timeout value cannot be unset/null");
     }
-    cql_duration duration = value_cast<cql_duration>(duration_type->deserialize(*timeout));
+    cql_duration duration = timeout.deserialize<cql_duration>(*duration_type);
```

Comment:
There is redundancy in supplying both the run-time type and the compile-time type. It is both inefficient, and can blow up if we supply conflicting types. Not suggesting to do anything about now, but keep it in mind for future improvement.